Topic Wise Questions with Solutions/Array/minimize_the_jump.py

```diff
@@ -0,0 +1,40 @@
+#Find Minimum number of jumps to reach end
+def min_jump(arr, n):
+    if n <= 1: 
+        return 0 #return zero values if the array less than one element
+    
+    #assigning the value 0    
+    left = 0
+    right = arr[0]
+    count = 1
+
+    while right < n-1:
+        count += 1 #incresing the value 
+        jump = max(i + arr[i] for i in range(left, right + 1)) #by this checking all the arr and finding min values 
+        left, right = right, jump #swaping the values
+    return count
+
+arr = list(map(int, input("Enter the elements: ").split()))
+size = len(arr)
+print('Minimum number of jumps to reach',
+      'end is', min_jump(arr, size))
+
+'''
+Time Complexity : O(N)
+Space Complexity : O(1)
+
+INPUT:
+    Enter the elements: 1 3 5 8 9 2 6 7 6 8 9
+
+OUTPUT  
+    Minimum number of jumps to reach end is 3
+
+Output: 3 (1-> 3 -> 8 -> 9)
+Explanation: Jump from 1st element 
```

Comment:
It's a minute change but I actually didn't understand the explanation at once....
Can you please write something like:  The first element in the list is 1 so only one jump is allowed like only if you wish to....The code is fine....But just a request to write a bit about the question in the beginning that's it

Reply:
Because I am unable to understand the question at one go

Reply:
Sure @sayantani11 please check my file I modified  as you suggested